plugins/module_utils/network/ios/rm_templates/snmp_server.py

```diff
@@ -320,7 +320,7 @@ def __init__(self, lines=None, module=None):
                 (\sversion\s(?P<version>1|3|2c))?
                 (\s(?P<version_option>auth|noauth|priv))?
                 (\svrf\s(?P<vrf>\S+))?
-                (\s(?P<community_string>\w+))?
+                (\s(?P<community_string>[-\w]+))?
```

Comment:
@KB-perByte Is this the only special character allowed? If not, can we make this regex pattern more generic?

Reply:
yes that is the only special character allowed, and have to collect just a word in the scope